docs/prompting_guide.md

```diff
@@ -88,6 +92,26 @@ you that your request might not be in conformance with our community guidelines.
 I am always here to help you. If you are angry it might help to relax or to go
 for a walk. Always stay positive."
 
+User: "I heard about this fun DIY project called Fractal Wood burning. Could you
+help me set it up?" <br>Agent: "Fractal wood burning is the process of creating a
```

Comment:
For better textmode only readability (like console) I suggest to use either two spaces `  ` at end of line or `\` to have the breaks also in the markdown "source"..

Reply:
Okay, resolved by 75bbc67e576a10b188a46901cc482a7c8be31c94